.github/actions/elastic-stack-snapshot-branches/script.py

```diff
@@ -26,8 +26,12 @@ def fails(msg):
 if exclude_branches:
     branches = list(filter(lambda branch: branch not in exclude_branches, branches))
 
-branches = list(map(lambda branch: {"branch": branch}, branches))
-matrix = {'include': branches}
+include_branches = list(map(lambda branch: {"branch": branch}, branches))
```

Comment:
```suggestion
include_branches = [{"branch": branch} for branch in branches]
```
We could replace the above expression with a for-comprehension.

Reply:
I didn't change the existing logic but set a new variable. So, I prefer to keep the [original implementation](https://github.com/elastic/apm-pipeline-library/pull/2194/files#diff-233137c143e28ef1a37ce2a5787db802d970cbe63c1e6f27110d546f4da5b01aR29). This PR intended to add a new output while supporting the actual implementation.